packages/docs/docs/sidekicks-quickstart.md

```diff
@@ -8,10 +8,17 @@ import Step3Profit from '../static/img/step-3-profit.png'
 
 # Quickstart: Fixie Sidekicks
 
-This is a quickstart guide to building and deploying a Fixie Sidekick. A sidekick is an embeddable, conversational
-assistant built using AI.JSX. Sidekicks harness the power of [DocsQA](/docsQA), [Tools](/tools), and [GenUI](/genUI) and bring everything together in a seamless experience.
+One of the best use cases for AI.JSX is to build a **Sidekick**, an AI-powered chatbot that is embedded in a web page or app and is able to answer questions, call APIs,
+and more. AI.JSX makes it easy to build Sidekicks with a rich UI, access to
+documents, and the ability to fetch live data and take action via API calls.
 
-_(TODO -> Matt, I'm thinking we might want to remove this last sentence since those pages are not yet built out. Or we can populate them with content.)_
+Sidekicks can be built and deployed in minutes. This quickstart will walk you through
+the entire process of building and deploying your own Sidekick.
+
+This tutorial relies on both AI.JSX as well as the [Fixie](https://fixie.ai) cloud
+platform, which provides a suite of APIs and tools for hosting and managing
+Sidekicks. It is possible to build and deploy Sidekicks without Fixie, but
+using Fixie makes the process much easier.
 
 :::info What You Will Do
 At the end of this quickstart you will:
```

Comment:
I know it's not in this diff, but `Prereq's installed` is a typo. Prereqs is not possessive.

Reply:
Right it's not possessive. "Prereq's" is a commonly used abbreviation of the full word.

Reply:
heh, ok, I buy the abbreviation angle. :smile: